### PR TITLE
Defend clear situation

### PR DESCRIPTION
**Pull request by Jason27chan**
## Description
Play for the defend clear situation. Mimicks basic122 offense. One robot attempting to pivot kick. Two robots defending the closest opponents to our goal

## Associated Issue
Issue #1346

### Test Case 1
1. Check that running the play leads to expected behavior

### CODE REVIEW

**Review by HussainGynai — APPROVED** [review]

**Review by HussainGynai — COMMENTED** [review]
Merge staging and use make pretty lines to fix circleCI checks.

File: soccer/gameplay/tactics/wall.py [hunk 1]
```diff
@@ -19,12 +19,12 @@ class State(enum.Enum):
     def __init__(
             self,
             num_defenders=3,  # number of defenders we're making the wall with (default 3)
-            curvature=.3,  # 'curvature' (in radians) of the wall 
+            curvature=0,  # 'curvature' (in radians) of the wall
```
Comment (tjones320) [inline soccer/gameplay/tactics/wall.py]: Why did you change these values for this commit?

File: soccer/gameplay/plays/DefendClear.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+        best_opps_to_mark = self.closest_opps_to_our_goal(num_supports)
+        self.add_subbehavior(skills.pivot_kick.PivotKick(), 'main')
+        for i in range(num_supports):
+            mark_skill = skills.mark.Mark()
+            mark_skill.mark_robot = best_opps_to_mark[i]
+            self.add_subbehavior(mark_skill, 'mark' + str(i))
+
+    ## Returns the robots that are closest to our goal
+    #
```
Comment (tjones320) [inline soccer/gameplay/plays/DefendClear.py]: Are you making sure to add your score function?

**Review by tjones320 — CHANGES_REQUESTED** [review]

File: soccer/gameplay/plays/DefendClear.py (hunk 2 above)
Comment (Jason27chan) [reply soccer/gameplay/plays/DefendClear.py]: Chris said to leave it blank because we'll be working on it later

Comment (Jason27chan) [reply soccer/gameplay/plays/DefendClear.py]: Just kidding. You were right; we needed to add it

File: soccer/gameplay/tactics/wall.py (hunk 1 above)
Comment (Jason27chan) [reply soccer/gameplay/tactics/wall.py]: youre right this is irrelevant to this PR i'll remove the changess

**Review by tjones320 — APPROVED** [review]
